Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM debian:jessie
+
+RUN echo "Upgrading distro..." && \
+    apt-get update > /dev/null && \
+    apt-get upgrade -y > /dev/null
+
+RUN echo "Installing dependencies..." && \
+    apt-get install -y git curl libcurl4-gnutls-dev sqlite3 libsqlite3-dev > /dev/null && \
+    apt-get install -y libtar libtar-dev zip libzip-dev unzip > /dev/null && \
+    apt-get install -y python python-distribute python-virtualenv python-setuptools > /dev/null && \
+    easy_install pip && pip install psycopg2 pymysql
+
+RUN echo "Cleaning up..." && \
```

Comment:
The cleanup commands need to be in the same `RUN` directive as the installation commands or else they're pointless (because any files in the image at the end of the `RUN` will remain in the final image even if a later directive deletes them).  See also https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#run